tapdb/multiverse_cache.go

```diff
@@ -271,6 +475,7 @@ func (r *rootNodeCache) cacheRoots(q universe.RootNodesQuery,
 
 	rootIndex := r.rootIndex.Load()
 	for _, rootNode := range rootNodes {
+		rootNode := rootNode
```

Comment:
FWIW, with the latest version of Go, we should no longer need to do this reassignment trick.